### PR TITLE
user doc: Updates for OpenAPI 3 support

### PR DESCRIPTION
**Pull request by MelissaFlinn**
This content is ready to merge.  Updates made to address the OpenAPI 3 support and changing "swagger" to "openapi". 
Google doc summary is here: https://docs.google.com/document/d/1ode8pCv03iiNCoGzOGvYIV6N9KLo87HZX5UH56XQCno

### CODE REVIEW

File: doc/modules/integrating-applications/c_overview-benefit-api-provider-integrations.adoc [hunk 1]
```diff
@@ -4,71 +4,71 @@
 [id='overview-benefit-api-provider-integrations_{context}']
 = Benefit, overview, and workflow for creating API provider integrations
 
-An API provider integration starts with a REST API service. 
-This REST API service is defined by an OpenAPI 2.0
-document that you provide when you create an API provider integration. 
+An API provider integration starts with a REST API service.
+This REST API service is defined by an OpenAPI (or 2)
```
Comment (TovaCohen) [inline doc/modules/integrating-applications/c_overview-benefit-api-provider-integrations.adoc]: Typo. "3" is missing.

File: doc/modules/integrating-applications/p_create-api-provider-integration.adoc [hunk 2]
```diff
@@ -32,16 +32,19 @@ to add to an operation's flow.
...
+* If you need to define the OpenAPI document, select *Create a new OpenAPI 3.x document* or *Create a new OpenAPI 2.x document*.
 
 . Click *Next*.
 +
 * If you uploaded a document, review or edit it:
 +
 .. Click *Review/Edit* to open the API Designer editor.
 .. Review and edit as needed.
++
+Optionally, if your document uses the OpenAPI 2 specification, you can click *Convert to OpenAPI 3* if you want the API Designer to convert your document to the OpenAPI 3 specification.
```
Comment (TovaCohen) [inline doc/modules/integrating-applications/p_create-api-provider-integration.adoc]: I think that "...to convert your document to conform with the OpenAPI 3 specification." 
would be better. Sorry I didn't notice this before.

Comment (MelissaFlinn) [reply doc/modules/integrating-applications/p_create-api-provider-integration.adoc]: I agree - thank you!

File: doc/modules/integrating-applications/c_overview-benefit-api-provider-integrations.adoc (hunk 1 above)
Comment (MelissaFlinn) [reply doc/modules/integrating-applications/c_overview-benefit-api-provider-integrations.adoc]: fixed - thanks

**Comment by TovaCohen** [comment]
Looks good! I'll merge this now.

**Pull request merged**